.github/workflows/test.yaml

```diff
@@ -38,5 +38,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/actions/sam-build
-      - run: sam validate --lint --template .aws-sam/build/template.yaml
+      - uses: ./.github/actions/sam-build-and-lint
```

Comment:
doesn't `sam validate` require access to AWS?  if so it probably doesn't work with a pre-merge workflow?

Reply:
`sam validate` needs access to AWS, but `sam validate --lint` doesn't. 🤷🏼